pyaedt/icepak.py

```diff
@@ -3182,3 +3182,446 @@ def get_face_normal(obj_face):
+        ht_correlation_type : str, optional
+            Choose between "Natural Convection" or "Forced Convection" for the correlation option.
+            Relevant if ``ht_correlation=True``. The default is ``"Natural Convection"``.
+        ht_correlation_fluid : str, optional
+            Fluid used for the correlation option. Relevant if ``ht_correlation=True``. The default is ``"air"``.
+        ht_correlation_flow_type : str, optional
+            Type of flow used for the correlation option. The choice is between ``"Turbulent"`` and ``"Laminar"``.
+            Relevant if ``ht_correlation=True``. The default is ``"Turbulent"``.
+        ht_correlation_flow_direction : str, optional
+            Flow direction used for the correlation option. Relevant if ``ht_correlation_type="Forced Convection"``.
+            The default is ``"X"``.
+        ht_correlation_value_type : str, optional
+             Choose between "Average Values" and "Local Values" for the forced convection correlation option.
+             Relevant if ``ht_correlation_type="Forced Convection"``. The default is ``"Average Values"``.
+        ht_correlation_free_stream_velocity : str or float, optional
+             Free stream flow velocity. If a float is used, m/s will be the default unit.
+             Relevant if ``ht_correlation_type="Forced Convection"``. The default is ``"1m_per_sec"``.
+        ht_correlation_surface : str, optional
+            Choose between "Top", "Bottom" and "Vertical" for the natural convection correlation option.
+            Relevant if ``ht_correlation_type="Natural Convection"``. The default is ``"Vertical"``.
+        ht_correlation_amb_temperature : str or float, optional
+            Ambient temperature for the natural convection correlation option.
+            Relevant if ``ht_correlation_type="Natural Convection"``. If a float is used, cel will be the default unit.
+            The default is ``"AmbientTemp"``.
+        shell_conduction : bool, optional
+            Whether to use the shell conduction option. The default is ``False``.
+        ext_surf_rad : bool, optional
+            Whether to use the external surface radiation option. Relevant if
+            ``ext_condition="Heat Transfer Coefficient"``. The default is ``False``.
+        ext_surf_rad_material : str, optional
+            Surface material for the external surface radiation option. Relevant if
+            ``ext_surf_rad=True``. The default is ``"Stainless-steel-cleaned"``.
+        ext_surf_rad_ref_temp : str or float, optional
+             Reference temperature for the external surface radiation option. Relevant if
+            ``ext_surf_rad=True``.  If a float is used, cel will be the default unit. The default is ``"AmbientTemp"``.
+        ext_surf_rad_view_factor : str or float, optional
+            View factor for the external surface radiation option. The default is ``"1"``.
```

Comment:
```suggestion
        ht_correlation_type : str, optional
            The correlation type for the heat transfer coefficient. Options are
            "Natural Convection" and "Forced Convection". This parameter is
            relevant if ``ht_correlation=True``. The default is ``"Natural Convection"``.
        ht_correlation_fluid : str, optional
            Fluid for the correlation option. This parameter is relevant if
            ``ht_correlation=True``. The default is ``"air"``.
        ht_correlation_flow_type : str, optional
            Type of flow for the correlation option. This parameter
            is relevant if ``ht_correlation=True``.  Options are ``"Turbulent"``
            and ``"Laminar"``. The default is ``"Turbulent"``.
        ht_correlation_flow_direction : str, optional
            Flow direction for the correlation option. This parameter is relevant if
            ``ht_correlation_type="Forced Convection"``. The default is ``"X"``.
        ht_correlation_value_type : str, optional
             Correlation value type for the forced convection correlation option.
             This parameter is relevant if ``ht_correlation_type="Forced Convection"``.
             Options are "Average Values" and "Local Values". The default is ``"Average Values"``.
        ht_correlation_free_stream_velocity : str or float, optional
             Free stream flow velocity. If a float value is specified, the default unit is m/s.
             This parameter is relevant if ``ht_correlation_type="Forced Convection"``.
             The default is ``"1m_per_sec"``.
        ht_correlation_surface : str, optional
            Correlation surface type for the natural convection correlation option.
            This parameter is relevant if ``ht_correlation_type="Natural Convection"``.
            Options are "Top", "Bottom", and "Vertical". The default is ``"Vertical"``.
        ht_correlation_amb_temperature : str or float, optional
            Ambient temperature for the natural convection correlation option.
            This parameter is relevant if ``ht_correlation_type="Natural Convection"``.
            If a float value is used specified, the default unit is cel.
            The default is ``"AmbientTemp"``.
        shell_conduction : bool, optional
            Whether to use the shell conduction option. The default is ``False``.
        ext_surf_rad : bool, optional
            Whether to use the external surface radiation option. This parameter
            is relevant if ``ext_condition="Heat Transfer Coefficient"``. The 
            default is ``False``.
        ext_surf_rad_material : str, optional
            Surface material for the external surface radiation option. This parameter
            is relevant if ``ext_surf_rad=True``. The default is ``"Stainless-steel-cleaned"``.
        ext_surf_rad_ref_temp : str or float, optional
             Reference temperature for the external surface radiation option. This paramter
             is relevant if  ``ext_surf_rad=True``.  If a float value is specified, the default unit is cel.
             The default is ``"AmbientTemp"``.
        ext_surf_rad_view_factor : str or float, optional
            View factor for the external surface radiation option. The default is ``"1"``.
```
For all occurrences of the following, it seems that "cel" should either be written out as Celcius or tagged as a literal string: If a float value is specified, the default unit is cel,